### DISCUSSION TITLE
안녕하세요, 제페토 샘플 월드 사용 관련해서 문의드립니다.

### DISCUSSION THREAD

**Discussion by IndiaInk10** [naverz/zepeto-studio-kor#1004] [5. Q&A]
월드를 제작할때, 샘플씬(캠핑장, 교실, 벗꽃공원 등)을 템플릿으로 사용해서 어느정도의 변형과 배치를 통해서 새로 출시할 수 있을지 궁금해서 한번 여쭤봅니다! 
만약에 가능하다면, 몇퍼센트 정도 유사도가 허용될지 정도로 예시를 주시면 될 것 같습니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 저희가 공식으로 제공한 샘플은 그대로 사용하셔도 무방합니다. (100% 똑같아도 됩니다)
제페토 활용 컨텐츠 가이드라인 참고 부탁 드립니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/595
감사합니다.
